scripts/npm/export-v2-translations.mjs

```diff
@@ -0,0 +1,80 @@
+import { stringify } from "csv";
+import { readFile, writeFile, mkdir } from "fs/promises";
+import { parseArgs } from "node:util";
+import { join, dirname, resolve } from "node:path";
+import { fileURLToPath } from "node:url";
+import { cwd } from "node:process";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const {
+  values: { "output-file": outputFile },
+} = parseArgs({
+  options: {
+    "output-file": {
+      short: "f",
+      type: "string",
+      default: "output/v2-translations.csv",
+    },
+  },
+});
```

Comment:
Remove the -f option and output to stdout.

Reply:
@louischan-oursky 

If running it by npm, there will be some logs in stdout, which pollute the output file.
```
> node@0.1.0 export-v2-translations
> node export-v2-translations.mjs
```

Reply:
Try `npm run -s <script>`

Reply:
Then the script must run with specific argument to work, which seems a bit weird?

Reply:
I am very surprised that npm run its output to stdout, not stderr.